src/raspbian/12/helix/arm32v6/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN git config --global user.email builder@dotnet-buildtools-prereqs-docker && \
     git clone --depth 1 --single-branch https://github.com/dotnet/arcade /scripts
 
 # Build the rootfs
-RUN /scripts/eng/common/cross/build-rootfs.sh armv6 raspbian --skipunmount
+RUN /scripts/eng/common/cross/build-rootfs.sh armv6 bookworm lldb15 --skipunmount
```

Comment:
@am11 we can probably remove the code in https://github.com/dotnet/arcade/blob/d1bc0779143e410ec82696f9865c4eb407db8d67/eng/common/cross/build-rootfs.sh#L170-L183 which sets the __UbuntuRepo, __CodeName etc since that is handled by the `bookworm` argument